Header header
string[] uri
float32[] velocity
float32[] acceleration
float32[] current
int8[] back_emf_sensing_state
float32[] back_emf
float32[] braking